configure.ml

```diff
@@ -842,9 +842,9 @@ let install = [
   "COQLIBINSTALL", "the Coq library", !prefs.libdir,
     Relative "lib", Relative "lib/coq", Relative "";
   "CONFIGDIR", "the Coqide configuration files", !prefs.configdir,
-    Relative "config", Absolute "/etc/xdg/coq", Relative "ide/coqide";
+    Relative "config", Absolute "/etc/coq", Relative "etc";
```

Comment:
This seems wrong on several levels.

1. As the second parameter says, this is the location of `"the Coqide configuration files"`. So, at the very least, this sentence should be changed.

2. The last parameter is the one used in `-local` mode, as far as I know. I do not understand why you are modifying it. Did you try Coqide after this change?

Reply:
w.r.t. 1, I had the feeling that coqrc was a coq thing, then it was made a coqide thing with the xdg stuff, and this PR is going back to it's original state, and the doc should be probably fixed too. (the equation coq=coqide is less and less true, so I'm in favor of this trajectory).

I concur that 2 is an oversight, but `git grep Configure.configdir` does not seem to be actually used by Coqide
```shell
gares@ollypat:~/COQ/master$ git grep configdir
...
ide/coqide/minilib.ml:  Envars.configdir ()
gares@ollypat:~/COQ/master$ git grep coqide_default_config
ide/coqide/minilib.ml:let coqide_default_config_dir () =
ide/coqide/minilib.ml:  [coqide_default_config_dir ()]
ide/coqide/minilib.mli:val coqide_default_config_dir : unit -> string
gares@ollypat:~/COQ/master$ git grep coqide_config_di
ide/coqide/minilib.ml:let coqide_config_dirs () =
ide/coqide/minilib.mli:val coqide_config_dirs : unit -> string list
```

Reply:
So, are you saying that `-configdir` is being used by absolutely nobody currently? Cannot we just scrap it then rather than repurpose it?

Reply:
Hum, yes, and I'm a bit puzzled. @LasseBlaauwbroek please double check my grep did not fail me.

Yes, it's better to add a new one rather than give a meaning to something that was ignored (and hence could have held crap and nobody would have notice). But I'm not so sure we need it to be configurable then, $prefix/etc/coq seems good for both opam and other packages no? (we may still need -local to do something, I talk about having configure option, which seems unneeded)